### PR TITLE
Shorten splash screen message

### PR DESCRIPTION
**Pull request by kjhenner**

### CODE REVIEW

File: templates/rc.local.erb [hunk 1]
```diff
@@ -25,16 +25,13 @@ echo \
...
 For the best experience please log in via SSH
 
     IP: $IP_ADDRESS
     username: root
     password: $PASSWORD
 
-Your password was randomly generated from international place names
-It is stored in /var/local/password and will continue to appear on this screen
-
+The password can be found in /var/local/password as well as this screen.
```
Comment (binford2k) [inline templates/rc.local.erb]: I'd rather it said something about being randomly generated, even if it were just a couple words.

**Comment by kjhenner** [comment]
@binford2k How does that look?

**Comment by binford2k** [comment]
I mean a word or two to explain why the password looks weird. :)

File: templates/rc.local.erb [hunk 2]
```diff
@@ -25,17 +25,14 @@ echo \
 
 <%= @login_message %>
 
-
-For the best experience please log in via SSH
+For the best experience please log in via SSH with the generated password below
```
Comment (samuelson) [inline templates/rc.local.erb]: Please log in via SSH with the randomly generated password below?

File: templates/rc.local.erb [hunk 3]
```diff
@@ -25,17 +25,14 @@ echo \
...
     IP: $IP_ADDRESS
     username: root
     password: $PASSWORD
 
-Your password was randomly generated from international place names
-It is stored in /var/local/password and will continue to appear on this screen
-
-To reset password, run the command: reset_password.rb
+The password can be found in /var/local/password as well as this screen.
+To generate and set a new password, run the command: reset_password.rb
```
Comment (samuelson) [inline templates/rc.local.erb]: To generate and set a new random password, run: reset_password.rb?

**Pull request merged**